rdflib/namespace/__init__.py

```diff
@@ -239,6 +239,16 @@ def __dir__(cls) -> Iterable[str]:
         values = {cls[str(x)] for x in cls.__annotations__}
         return values
 
+    def jsonld_context(self, pfx: str) -> str:
+        """This function creates a JSON-LD 'context' JSON object from this
+        DefinedNamespace namespace and it's members"""
+        terms = {pfx: str(self._NS)}
+        for key, term in self.__annotations__.items():
+            if isinstance(term, URIRef):
```

Comment:
Tests will pass when this is changed to
```suggestion
            if issubclass(term, URIRef):
```

Explanation here: https://github.com/RDFLib/rdflib/pull/1706#discussion_r801077815